js/script.js

```diff
@@ -126,12 +97,16 @@ document.addEventListener("DOMContentLoaded", () => {
                 `1. Reset your board and try again.\n` +
                 `  - Look for a little black button near the power port.\n` +
                 `2. Refresh your browser and try again.\n` +
-                `3. Double-check your board type and serial port selection.\n` +
-                `4. Post on the Support Forum (link above) with this info:\n\n` +
+                `3. Make sure you are not connected in another browser tab.\n` +
+                `4. Double-check your board type and serial port selection.\n` +
+                `5. Post on the Support Forum (link above) with this info:\n\n` +
                 `"Firmware Tool: ${e}"\n`
             );
-            disconnect();
-        }
+            if (espStub) {
+              await espStub.disconnect();
```

Comment:
Should this call the local `disconnect()` method?

Reply:
Yes, probably.